UI/window-basic-main.hpp

```diff
@@ -225,6 +225,7 @@ class OBSBasic : public OBSMainWindow {
 
 private:
 	obs_frontend_callbacks *api = nullptr;
+	void OnEvent(enum obs_frontend_event event);
```

Comment:
Probably should be put with the other methods.

Reply:
Fixed